spec/lib/dispatch-rider/notification_services/base_spec.rb

```diff
@@ -12,24 +12,25 @@ def publish(msg)
   end
 
   before :each do
-    DispatchRider::NotificationServices::Base.any_instance.stub(:notifier_builder).and_return(OpenStruct)
-    DispatchRider::NotificationServices::Base.any_instance.stub(:channel_registrar_builder).and_return(DispatchRider::Registrars::SnsChannel)
-    DispatchRider::NotificationServices::Base.any_instance.stub(:channel) do |name|
+    allow_any_instance_of(described_class).to receive(:notifier_builder).and_return(OpenStruct)
+    channel = DispatchRider::Registrars::SnsChannel
+    allow_any_instance_of(described_class).to receive(:channel_registrar_builder).and_return(channel)
+    allow_any_instance_of(described_class).to receive(:channel) do |name|
       subject.notifier.topics[subject.fetch(name)] if name == :foo
     end
   end
 
   subject do
-    DispatchRider::NotificationServices::Base.new({:topics => {}})
+    DispatchRider::NotificationServices::Base.new({ :topics => {} })
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.